__tests__/app.test.js

```diff
@@ -0,0 +1,42 @@
+const db = require("../db/connection");
+const app = require("../app");
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/");
+const { describe, test, expect } = require("@jest/globals");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/topics", () => {
+  test("GET 200: should return an array of topics objects ", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        expect(response.status).toBe(200);
+        expect(response.body.topic.length).toBeGreaterThan(0);
+        expect(Array.isArray([response])).toBe(true);
```

Comment:
You don't want to do be doing this. You've actually hard-coded a passing assertion here by wrapping the object `response` in an array. What I think you were aiming for here is checking the returned list of topics to be an array. Either way you don't need to check that what comes back is an array because you then do a `forEach` which can only work on an array